contracts/pair_stable_bluna/src/contract.rs

```diff
@@ -1476,14 +1457,19 @@ pub fn migrate(deps: DepsMut, env: Env, _msg: MigrateMsg) -> Result<Response, Co
     match contract_version.contract.as_ref() {
         "astroport-pair-stable" => match contract_version.version.as_ref() {
             "1.0.0" => {
-                let config = CONFIG.load(deps.storage)?;
+                let mut config = CONFIG.load(deps.storage)?;
+                config.bluna_rewarder = addr_validate_to_lower(deps.api, &msg.bluna_rewarder)?;
+                config.generator = addr_validate_to_lower(deps.api, &msg.generator)?;
+                CONFIG.save(deps.storage, &config)?;
                 response
                     .messages
                     .push(get_bluna_reward_holder_instantiating_message(
                         deps.as_ref(),
                         &env,
                         &config.factory_addr,
-                    )?)
+                    )?);
+
+                BLUNA_REWARD_GLOBAL_INDEX.save(deps.storage, &cosmwasm_std::Decimal256::zero())?;
```

Comment:
is we need to set BLUNA_REWARD_GLOBAL_USER_INDEX to zero?

Reply:
Yes, otherwise we will get an error on reading it

Reply:
Replaced it with better approach